packages/nimble-components/src/wafer-map/index.ts

```diff
@@ -211,12 +230,19 @@ export class WaferMap extends FoundationElement {
             } else if (this.waferMapUpdateTracker.requiresDrawnWaferUpdate) {
                 this.renderer.drawWafer();
             }
-            this.eventCoordinator.attachEvents();
+            this.zoomHandler.connect();
         } else if (this.waferMapUpdateTracker.requiresRenderHoverUpdate) {
             this.renderer.renderHover();
         }
     }
 
+    /**
+     * @internal
+     */
+    public isExperimentalRenderer(): boolean {
```

Comment:
I left this because we are also checking the type of the dies table. When using the function it will consider that undefined is still part of the type

Reply:
ahh, makes sense, it's not related to render modes but actually inspecting table data. Makes perfect sense!